src/v/cluster/members_backend.cc

```diff
@@ -485,18 +511,44 @@ members_backend::try_to_finish_update(members_backend::update_meta& meta) {
         co_return;
     }
 
+    auto is_finished = [](const auto& r) {
```

Comment:
This is pulled out from below as we use it in 2 places now (the new one being the log line below).